packages/web3-utils/src/utils.js

```diff
@@ -534,8 +534,29 @@ var toNumber = function(value) {
     return typeof value === 'number' ? value : hexToNumber(toHex(value));
 }
 
+// 1.x currently accepts 0x... strings, bn.js after update doesn't. it would be a breaking change
```

Comment:
Thinking about this now, I dont think we should be changing how BN is being created... the [test case](https://github.com/ChainSafe/web3.js/pull/5074/files#diff-bd97e9c608060469030c1d6a655baa533c0aa0b11939b0670ebdb41aa51d786bR663) is working fine and this is adding an extra feature this PR should just be focusing on upgrading and fixing `BN`.

Reply:
You are right about the scope, but, shouldn't we handle the <change> introduced too? (you mean to do it in another PR? )